sherpa/models/model.py

```diff
@@ -46,12 +46,12 @@ def cache_model(cls, pars, xlo, *args, **kwargs):
         digest = ''
         if use_caching:
 
-            data = [ numpy.array(pars).tostring(), str(kwargs.get('integrate',0)), 
+            data = [ numpy.array(pars).tostring(), kwargs.get('integrate', b'0'),
```

Comment:
Hmm... you are probably right. I'll try and see if there is a test passing `integrate` as a boolean. Probably there is not. Or I am missing something :)